CodeStarterCamp_Week2/main.swift

```diff
@@ -1,12 +1,19 @@
-func printLottoResult(matchNumbers: Set<Int>) {
-    if matchNumbers.isEmpty {
-        print("아쉽지만 겹치는 번호가 없습니다.")
+var roundNumbers: Dictionary<String, Set<Int>> = [String: Set<Int>]()
+
+func searchWinningNumber (round: Int) {
+    if let winningNumber = roundNumbers["\(round)회차"] {
+        print("\(round)회차의 로또 당첨 번호는 \(numbersToStr(numbers: winningNumber)) 입니다.")
     } else {
-        print("축하합니다! 겹치는 번호는 \(matchNumbers.map{String($0)}.joined(separator: ", ")) 입니다!")
+        print("\(round)회차는 아직 추첨되지 않았습니다.")
     }
 }
 
-let myLottoNumbers = [1, 2, 3, 4, 5, 6]
-let lottoNumbers = makeLottoNumber()
+func numbersToStr (numbers: Set<Int>) -> String{
```

Comment:
```suggestion
func numbersToStr(numbers: Set<Int>) -> String {
```

str같은 축약어는 지양하는게 좋습니다!
파라미터 명도 중복되는 정보라 자연스럽게 읽히는 인자 라벨을 사용해봐도 좋을 것 같습니다.